src/main/java/emu/grasscutter/command/commands/SetConstCommand.java

```diff
@@ -30,6 +29,25 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
             return;
         }
 
+        if (args.get(0).equalsIgnoreCase("all")) {
+            if (args.size() < 2) {
+                sendUsageMessage(sender);
+                return;
+            }
+            try {
+                int constLevel = Integer.parseInt(args.get(1));
```

Comment:
Consider using `args.remove(index)` to avoid duplicating the level parsing.
Alternatively, move the "all" to the second argument and take inspiration from the `switch` handling in other commands.